### ISSUE TITLE
Do you support GTD (Good-til-Date/Time) orders?

### ISSUE THREAD

**Issue by P9avel** [ArjunVachhani/order-matcher#47]
Hi, do you support GTD (Good-til-Date/Time) orders?

**Comment by ArjunVachhani** [comment]
Yes, order-matcher supports Good-till-Date/time orders. You can set `Order.CancelOn` property to auto cancel in the future. `CancelOn` is represented as seconds from Jan 1, 1970 UTC (EPOCH seconds).

Checkout [Order Wiki](https://github.com/ArjunVachhani/order-matcher/wiki/2.-Order) to learn about Order Type fields. and various supported order types.

**Comment by P9avel** [comment]
Yes, all OK, sorry for dummy question
